api/src/paths/project/{projectId}/update.ts

```diff
@@ -249,15 +249,25 @@ PUT.apiDoc = {
               }
             },
             location: {
-              description: 'The project location object',
+              title: 'Location',
               type: 'object',
-              required: ['geometry'],
```

Comment:
I think geometry is still required?

Reply:
should double check the others too, especially if they are required by the Database.

Reply:
The database doesn't require it, but our front-end does.